### ISSUE TITLE
Написать исследовательскую часть

### ISSUE THREAD

**Issue by wldmr-key** [sloyka-debug/kursovaya#1]
Все , что написано очень неплохо, но есть один очень большой минус, на мой взгляд - это постановка задачи

К первым трем абзацам претензий нет - все правильно, но после них ожидается конкретизация задачи - какая еда трактуется тобой, как домашняя, кто является целевой группой твоего приложения (ну, про едоков все понятно, хотя их потребительские характеристики нуждаются в уточнении - ты же осуждаешь рестораны с минимальным чеком в 30 долларов, а как насчет производителей? Кто это? Ты там что-то дальше говоришь про самозанятых. С чего бы?)

То есть у тебя явно есть светлые мысли о назначении твоего приложения, но ты их зачем-то скрываешь ((

И вот эта недоговоренность во многом нивелирует твой хороший анализ аналогов, так как ты сразу не определила правила игры.

Кроме того, подробная постановка задачи и/или описание предметной области позволит тебе определить словарь, при помощи которого ты опишешь основные сценарии взаимодействия, сущности и атрибутя данных

**Comment by wldmr-key** [comment]
Месяц прошел... Прогресс есть?

**Comment by sloyka-debug** [comment]
Текст залит, надо ли что-то менять?

P.S: оповещения не было, и сообщение от 9 ноября я увидела только сейчас :D

**Comment by wldmr-key** [comment]
да, куда уж менять.. Вообще-то графическую часть надо бы приделать...

У вас с Артемом получились уморительно похожими технологические части - примерно одинаковые драмы ))

И, да, я понимаю, что слово "идентификатор" очень длинное и писать его долго, но твой неологизм "айди" заслуживает порицания
